src/index.ts

```diff
@@ -13,11 +13,10 @@ const sendEvent = async (
   const requestBody = {
     event_source: payload.event_source || 'web',
     event_source_id: payload.properties.pixelCode || settings.pixelCode,
+    test_event_code: settings.testKey,
```

Comment:
In Production test_event_code need to be removed.
We can add this in README too.
Make sure to remove test_event_code before switching to 'Production' mode.
https://business-api.tiktok.com/portal/docs?id=1771100984456193

Reply:
Made a change so that it will be included only in case users pass it as a setting